grpc/sa-wrappers.go

```diff
@@ -566,17 +580,25 @@ func (sas StorageAuthorityServerWrapper) CountCertificatesByNames(ctx context.Co
 		return nil, err
 	}
 
-	resp := &sapb.CountByNames{}
-	for k, v := range byNames {
-		castedV := int64(v)
-		// Make a copy of k because it will be reassigned with each loop.
-		kCopy := k
-		resp.CountByNames = append(resp.CountByNames, &sapb.CountByNames_MapElement{Name: &kCopy, Count: &castedV})
+	resp := &sapb.CountByNames{
```

Comment:
Nit: Let's make this consistent with the one below (return directly, format all on one line)